src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -1109,6 +1123,7 @@ void improve(long maxBps)
                 if (!ENABLE_ONSTAGE_VIDEO_SUSPEND)
                 {
                     ratesIdx = 0;
+                    oversending = rates[0].bps > maxBps;
```

Comment:
Is the flag always cleared? I.e. if an endpoint is selected and oversending, and then gets unselected, the oversending flag should be reset to false, right?

Reply:
right, the TrackBitrateAllocation is ephemeral and gets destroyed after the bandwidth distribution decision.